app/controllers/foreman_salt/minions_controller.rb

```diff
@@ -6,14 +6,21 @@ class MinionsController < ::ForemanSalt::ApplicationController
     add_smart_proxy_filters :node, :features => 'Salt'
 
     def node
-      enc = {}
+      enc = {'foreman' => {}}
```

Comment:
Sorry, I've misinterpreted how the generated pillars were passed down to the salt master.
Most of salt users like me would expect just to see the pillars as seen by the minion. Now I understand that what is shown pressing "Salt ENC' is the raw data passed to the salt master.
Maybe better to just have a "pillar" button that shows the actual pillar data that minion gets.